### ISSUE TITLE
Fix Linux reading of older files

### ISSUE THREAD

**Issue by rmpowell77** [calband/calchart#468]
Notice that we do this in reading calchart 3.3 sheets (with the older parser):

```
            auto c = Coord(reader.Get<uint16_t>(), reader.Get<uint16_t>());
```

There's no guarantee of the execution order of reading the two values here.  Should be:

```
            auto x = reader.Get<uint16_t>();
            auto y = reader.Get<uint16_t>();
            auto c = Coord(x, y);
```

**Comment by rmpowell77** [comment]
only close when this has been pulled into mainline.

**Comment by rmpowell77** [comment]
Fixed with #470 and #473
